internal/interpreter/dynamic_component.rs

```diff
@@ -1497,32 +1524,33 @@ unsafe extern "C" fn get_item_ref(component: ComponentRefPin, index: usize) -> P
     }
 }
 
+fn parent_repeater<'a>(instance_ref: &InstanceRef<'a, 'a>) -> Option<ItemRc> {
+    let parent_item_index = instance_ref
+        .component_type
+        .original
+        .parent_element
+        .upgrade()
+        .and_then(|e| e.borrow().item_index.get().cloned());
+    if let (Some(parent_offset), Some(parent_index)) =
+        (instance_ref.component_type.parent_component_offset, parent_item_index)
```

Comment:
If the caller is expected to ensure that `instance_ref` is a child of a repeater, then ignore this comment :-).

Otherwise I think that you may want to check if the upgraded parent element's `repeated` field `is_some()` to ensure this is really the case.